packages/expo-screen-capture/src/ScreenCapture.ts

```diff
@@ -107,6 +108,44 @@ export function addScreenshotListener(listener: () => void): EventSubscription {
   return ExpoScreenCapture.addListener(onScreenshotEventName, listener);
 }
 
+// @needsAudit
+/**
+ * Adds a listener that will fire whenever the app starts or stops to be recorded.
+ *
+ * @param listener The function that will be executed when the user takes a screen recording.
+ * This function accepts a single boolean argument that indicates whether the screen recording
+ * started or stopped.
+ * @platform ios
+ * @example
+ * ```ts
+ * const [sensitiveContentVisible, setSensitiveContentVisible] = useState(true);
+ *
+ * useEffect(() => {
+ *   const subscription = ScreenCapture.addRecordingListener(({ isCaptured }) => {
+ *     if (isCaptured) {
+ *       // Hide sensitive content
+ *       setSensitiveContentVisible(false);
+ *     } else {
+ *       // Show sensitive content again
+ *       setSensitiveContentVisible(true);
+ *     }
+ *   });
+ *
+ *   // Clean up subscription when component unmounts
+ *   return () => {
+ *     subscription?.remove();
+ *   };
+ * }, []);
+ * ```
+ * @return A `Subscription` object that you can use to unregister the listener, either by calling
+ * `remove()` or passing it to `removeRecordingListener`.
```

Comment:
Afaik, we usually don't include usage examples in function documentation, the example in `screen-capture.mdx` is sufficient 😄